app/models/user.rb

```diff
@@ -15,7 +18,7 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :trackable, :validatable,
          :confirmable, :recoverable, :omniauthable, omniauth_providers: %i[facebook]
 
-  validates :username, presence: true
+  validates :username, presence: true, uniqueness: true
```

Comment:
We don't use username for login or something security important. It's ok to have a lot of the same named users in our case